nodejs/awsx/apigateway/README.md

```diff
@@ -98,6 +98,121 @@ A complete example can be found [here](https://github.com/pulumi/pulumi-awsx/blo
 
 API Gateway can perform basic validations against request parameters, a request payload or both. When a validation fails, a 400 error is returned immediately.
 
+### Lambda Authorizers
+
+[Lambda Authorizers](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-use-lambda-authorizer.html) are AWS Lambda functions that provide control access to an API. You can define a Lambda Authorizer for an Event Handler Route or a Static Route. API Gateway supports "request" or "token" type Lambda authorizers.
+
+To define an Authorizer, you provide a Lambda that fulfills `aws.lambda.EventHandler<AuthorizerEvent, AuthorizerResponse>` or you provide information on a pre-existing Lambda authorizer. The example below shows defining the Authorizer Lambda directly inline. See the [Event Handler Route](#Event-Handler-Route) section for other ways you can define a Lambda for the Authorizer.
+
+Below is an example of a custom "request" Lambda Authorizer.
+
+```ts
+import * as awsx from "@pulumi/awsx";
+
+const api = new awsx.apigateway.API("myapi", {
+    routes: [{
+        path: "/b",
+        method: "GET",
+        eventHandler: async () => {
+            return {
+                statusCode: 200,
+                body: "<h1>Hello world!</h1>",
+            };
+        },
+        authorizers: [{
+            authorizerName: "prettyAuthorizer",
+            parameterName: "auth",
+            parameterLocation: "query",
+            authType: "custom",
+            type: "request",
+            handler: async (event: awsx.apigateway.AuthorizerEvent) => {
+                console.log("Received event:", JSON.stringify(event, null, 2));
```

Comment:
so, from this example, it doesn't look like we're actually validating anything about the incoming data.  Should we potentially remove this console.log line and add something like:

```ts
// Perform domain-specific validation of the event
```

?